ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function checkCli() {
+    set +o errexit
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
+    command -v ${IBMCLOUD_CLI}
+    ${IBMCLOUD_CLI} --version
+    ${IBMCLOUD_CLI} plugin list
+    id
+    echo "Home: $HOME IBMCloud_HOME $IBMCLOUD_HOME"
```

Comment:
When some unexpected issues happened (or ibmcloud cli does not install or does not work), the `echo` will hide the issue, I think you need to return some non-zero value when some commands failed.